src/LibraryManager.Vsix/UI/Controls/TargetLocation.xaml.cs

```diff
@@ -270,6 +275,8 @@ private void TargetLocationSearchTextBox_TextChanged(object sender, TextChangedE
                     Flyout.IsOpen = true;
                 });
             }
+
+            InstallationFolder.DestinationFolder = TargetLocationSearchTextBox.Text;
```

Comment:
Do you still need this?

Reply:
Yes, this is still needed.

---
In reply to: [200753183](https://github.com/aspnet/LibraryManager/pull/285#discussion_r200753183) [](ancestors = 200753183)